api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java

```diff
@@ -339,13 +340,9 @@ private CohortDefinition getNumberOfPatientsInFichaClinicaWithViralLoadResult()
   private CohortDefinition
       getNumberOfPatientsTransferredInFromOtherHealthFacilitiesDuringCurrentMonth() {
 
-    EptsTransferredInCohortDefinition cd = new EptsTransferredInCohortDefinition();
-    cd.setProgramEnrolled(hivMetadata.getHIVCareProgram());
-    cd.setProgramEnrolled2(hivMetadata.getARTProgram());
-    cd.setPatientState(hivMetadata.getArtCareTransferredFromOtherHealthFacilityWorkflowState());
```

Comment:
@iteliolucas dont you need the workflow?

Reply:
Sure thing @psmatsinhe that's why I got rid of it under the last commit 'Swap to EptsTransferredInCohortDefinition2'

Reply:
> Sure thing @psmatsinhe that's why I got rid of it under the last commit 'Swap to EptsTransferredInCohortDefinition2'

Inside of EptsTransferredInCohortDefinitionEvaluator2 we do have a method named 'setupMetadata()' which takes care of Program Workflow States